Payments/UserInterface/06a_ListConfirmations.md

```diff
@@ -0,0 +1,213 @@
+In the left margin column, all available attribute groups are displayed. Click a group to display the attributes that are assigned to this group on the right side of the *Attributes* tab. 
+
+- ![Folders](../../Assets/Icons/Folders01.png "[Folders]") (Folders)  
+    Attribute group that contains attribute sub-groups. Click the attribute group or the arrow left to the attribute group to expand the group and display the attribute sub-groups. The following sub-groups are available:
+    - *[Payment service provider] Outgoing Payment Base Data*   
+      This folder contains attributes that are specific for the payment service provider.
+    - *Outgoing Payment Base Data*   
+      This folder contains general fields that Actindo needs to query payments data. These data support standard functions of Actindo. See the description of the fields above.
+    - *Unassigned Group*   
+      This folder contains attributes that are not assigned the other groups.
+
+- ![Folder](../../Assets/Icons/Folder01.png "[Folder]") (Folder)  
+    Attribute group. Click the attribute group to display all attributes that are assigned to the selected attribute group on the right side of the *Attributes* tab.
+
+The right side of the *Attributes* tab displays all attributes that are assigned to the selected group in the left margin column. Depending on the selected attribute group, the displayed fields may vary. The attributes are imported by the payment service provider driver. All fields are read-only.   
+
+- ![Collapse/Expand](../../Assets/Icons/CollapseExpand01.png "[Collapse/Expand]") (Collapse/Expand)    
+    Click this button to collapse or expand the left margin column with the attribute groups. When the left margin is expanded and you click this button, the column is collapsed. When the column is collapsed and you click this button, the column is expanded. This is available for the *Unassigned Group* folder only.
+
+
+## Confirmations &ndash; Dependencies
+
+![Dependencies](../../Assets/Screenshots/Payments/Confirmations/DependenciesConfirmation.png "[Dependencies]")
+
+Since a confirmation ticket has no dependencies to other entities in Actindo by default, this sub-tab is not relevant.
+
+<!---Marina, hier gibt es keine Logs-->
+
+## Create view
+*Payments > Confirmations > Tab LIST > Button Points > Menu entry create*
+
+![Create view](../../Assets/Screenshots/Payments/LIST/CreateView.png "[Create view]")
+
+- *Name*   
+    Enter a name for the view.
+
+- [CANCEL]   
+    Click this button to cancel creating a view. The *Create view* window is closed.
+
+- [SAVE]   
+    Click this button to save the new view and display it in the *VIEW* drop-down list. The *Create view* window is closed.
+
+
+
+## Rename view
+
+*Payments > Confirmations > Tab LIST > Button Points > Menu entry rename*
+
+![Rename view](../../Assets/Screenshots/Payments/LIST/RenameView.png "[Rename view]")
+
+- *Name*   
+    Click this field to edit the view name.
```

Comment:
Create view and Rename view: 
In der Regel beschreiben wir die Felder das erste Mal, das sie vorkommen (also hier in Authorizations). Danach nur Header, Pfad, Screenshot und Verweis. Deswegen gibt es danach in der ersten Datei (hier Authorizations) ein Rückverweis, damit man einfach zurückgehen kann). Beispiel: https://github.com/Actindo-AG/Documentation/blob/main/Fulfillment/UserInterface/01a_List.md#create-view

Reply:
Verstehe ich nicht, Wäre es nicht besser, in alle einen Verweis auf die Authorization  zu machen. In Fullfillment ist es übrigens überall drin. Da verstehe ich links schon gar nicht